packages/poseidon-cipher/package.json

```diff
@@ -0,0 +1,51 @@
+{
+    "name": "@zk-kit/poseidon-cipher",
+    "version": "0.1.0",
+    "description": "Poseidon encryption and decryption in TypeScript.",
+    "license": "MIT",
+    "keywords": [
+        "typescript",
+        "poseidon",
+        "cryptography",
+        "encryption",
+        "decryption",
+        "ciphertext",
+        "plaintext"
+    ],
+    "iife": "dist/index.js",
+    "unpkg": "dist/index.min.js",
+    "jsdelivr": "dist/index.min.js",
+    "main": "dist/index.node.js",
+    "exports": {
+        "import": "./dist/index.mjs",
+        "require": "./dist/index.node.js",
+        "types": "./dist/types/index.d.ts"
+    },
+    "types": "dist/types/index.d.ts",
+    "files": [
+        "dist/",
+        "src/",
+        "README.md",
+        "LICENSE"
+    ],
+    "repository": "git@github.com:privacy-scaling-explorations/zk-kit.git",
+    "homepage": "https://github.com/privacy-scaling-explorations/zk-kit/tree/main/packages/poseidon-cipher",
+    "scripts": {
+        "build:watch": "rollup -c rollup.config.ts -w --configPlugin typescript",
+        "build": "rimraf dist && rollup -c rollup.config.ts --configPlugin typescript",
+        "prepublishOnly": "yarn build"
+    },
+    "publishConfig": {
+        "access": "public"
+    },
+    "devDependencies": {
+        "@zk-kit/eddsa-poseidon": "workspace:^",
```

Comment:
I'm not 100% sure that NPM and Yarn resolve this version when the package is published. We can try tho.

Reply:
should I put the exact version?

Reply:
I think it's worth trying it. If it works we could adopt the same approach with the other packages too.

Reply:
Let's see 👀 eslint and everything else should be fixed now